src/main/java/com/exasol/swqa/maven/SummarizerMojo.java

```diff
@@ -140,10 +141,13 @@ private void writeSummaryFile(final Path mavenTargetPath, final float coverage)
 
     // [impl -> dsn~writing-code-coverage-value~1]
     private static String generateSummaryJSON(final float coverage) {
+        // We are enforcing the US locale for numbers here in order to make sure that a dot will be used as decimal
+        // separator.
+        final String coverageAsUSLocaleString = String.format(Locale.US, "%.1f", coverage);
```

Comment:
Optional: You could add a test that verifies correct json even when using a German locale.

Reply:
Done.